pkg/cmd/server/kubernetes/master.go

```diff
@@ -97,6 +105,22 @@ func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string
 	recycler.Run()
 }
 
+// attemptToLoadRecycler tries decoding a pod from a filepath for use as a recycler for a volume.
+// If successful, this method will set the recycler on the config.
+// If unsucessful, an error is returned.
+func attemptToLoadRecycler(path string, config *volume.VolumeConfig) error {
+	glog.V(5).Infof("Attempting to load recycler pod file from %s", path)
+	if path != "" {
```

Comment:
The code calling this func had just set a default recycler pod.  This is an attempt to allow CLI override if the file path is provided.